packages/babel-plugin-check-es2015-constants/test/fixtures/general/no-assignment/expected.js

```diff
@@ -0,0 +1,4 @@
+var MULTIPLIER = 5;
+MULTIPLIER = (function () {
```

Comment:
Definitely, I want to add executed tests as well to assert the runtime error is thrown. I need to look at how other packages do these tests first though, because everything I have tried so far did not work.